src/components/Header/SearchSuggestion.jsx

```diff
@@ -132,8 +132,22 @@ function SearchSuggestion({ setSearchCurrentValue = () => {} }) {
 
   const handleSearch = (event) => {
     const { key, type } = event;
+    const name = event.target.value;
     if (key === 'Enter' || type === 'click') {
-      navigate({ pathname: `/explore`, search: createSearchParams({ search: inputValue || '' }).toString() });
+      const splitName = name.split(':');
+      let inputInSuggestions = false;
+
+      for (let i = 0; i < suggestionData.length; i++) {
```

Comment:
this can be rewritten functionally:
```
const inputInSuggestions = suggestionData.some(sd => sd.repoName === splitName[0] && sd.tag === splitName[1])
```